blueflood-http/src/main/java/com/rackspacecloud/blueflood/outputs/handlers/HttpEventsQueryHandler.java

```diff
@@ -29,47 +30,37 @@ public HttpEventsQueryHandler(EventsIO searchIO) {
     @Override
     public void handle(ChannelHandlerContext ctx, HttpRequest request) {
 
-        if(request.getMethod().toString().equalsIgnoreCase("GET")) {
```

Comment:
@shintasmith 
I moved *ALL* OPTIONS requests to be handled by the new class HttpOptionsHandler in .../blueflood-http/.../outputs/handlers/HttpOptionsHandler.java.

See the code in class HttpMetricDataQueryServer for the handler changes to 
```
router.options("/v2.0/:tenantId/events/getEvents", new HttpOptionsHandler());
```

i.e.
Previously, this class was routing both GET and OPTIONS request for this path to HttpEventsQueryHandler, hence the need to check the request methods.  But now only GET requests are routed to this the HttpEventsQueryHandler so this class is now dedicated to only have to serve GET requests (similar to other handlers in the outputs.handlers package with the exception of the new HttpOptionsHandler class).